### DISCUSSION TITLE
IKEA cover Fyrtur integrated but cannot be addressed...

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#10219] [Devices]
Hi,

hopefully I am addressing my question at the right place - first time I am posting here ;)
I have searched through the discussion  but could find any solution - if I am mistaken, just please hint me to the right direction ;)

During the last two days I have setup Zigbee2Mqtt  and up to now I am pretty impressed. However, I have one problem right now, which I cannot get resolved:

I have integrated eight different IKEA Fyrtur covers and all of them got easily integrated and are showing up in Zigbee2Mqtt, however, only four of them are addressable, the other four are there but I cannot control them whatsoever.

I have integrated eight covers as follows: 
![image](https://user-images.githubusercontent.com/61102969/146634621-141ee022-948f-4098-b35c-c9c88020be8c.png)

At the beginning I thought that the battery status would I’ve me a hint. But some of them that do not show the battery status can be controlled and vice versa.

What I have found though was that the State content is different. While the working devices show such a State content:

`{
    "battery": 78,
    "linkquality": 255,
    "position": 84,
    "update": {
        "state": "available"
    },
    "update_available": true
}`

Whereas the non-working are somewhat different:

`{
    "linkquality": 255,
    "update": {
        "state": "available"
    },
    "update_available": true
}
`

... or very different:
`{
    "linkquality": 255
}`

All of them have been added too the Zigbee2Mqtt network the same way, same day, etc. but right half of them work and the other half won’t.

I am sure the error sits in front of the keyboard - anybody has any idea what I am doing wrong?

Thanks for any help!

P.S. I have posted this question also in the Discourse Forum just to find out that it has been depreciated - I hope it is ok that I repost it here ;)

**Comment by Eurymedon** [comment]
I've got nearly the same problem:
I can connect to 3 of 4 blinds. After paring the 4th blind, it works for some minutes, then the 4th blind stops working at all. It can't be controlled anymore. After some tries, 2 of 4 won't work. Seem's that it's some kind of "growing" problem.

Things I tried (without success): 
Deleting and pairing the blind again and again.
Restarting the container.
Deleting the blind, restarting the container, paring it again.

**Comment by Twit123** [comment]
I used to have the same problem. One or two of my 4 blinds would not work properly.

The solution I used was to delete and re-pair BUT I had the blind right next to the controller. A bit of a PITA but since I did that, all 4 have not given any issues in over 6 months.

Si

**Comment by Eurymedon** [comment]
Well - I experimented throughout the day.
I deleted all devices, paired them angain, restarted the zigbee2mqtt-container. Over and over again...
Moste times only one of the blinds won't react. When none react, I restarted the container - and tadaa, all blinds worked properly.

But just for just 2 moves. One down, one up - no reaction at all.
The log ran full of messages like this:
MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":159,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":151,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":151,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":151,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":90,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":90,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":151,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":100,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":45,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":90,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":95,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":100,"position":100}'
info MQTT publish: topic 'zigbee2mqtt/Rollo Wohnzimmer links', payload '{"battery":92,"linkquality":151,"position":100}'

But: No reaction of the blind. It can't be controlled at all. 

It's allways an other blind, when I delete, repair and reboot. Sometimes all three in one room work properly, but the one in the other room stops. Or one of the thre in the first room works and that one in the other doesn't. It's like randomly 1 of 4 just fails.

I can say, that there where many error messages, when I delete the blinds, that the deleted device is unknown until I pair it again. 
When I delete a lamp and repair it, that won't happen at all. The lamp is just gone and I can pair it again. 
When I delete a blind, it spams error messages, like it spams those position messages.
